<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabSudAuswahl</class>
 <widget class="QWidget" name="TabSudAuswahl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutFilter">
     <item>
      <widget class="FilterButtonSud" name="btnFilter">
       <property name="icon">
        <iconset theme="filter"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tbFilter">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Textfilter&lt;/span&gt; für die Sudsuche. Gesucht wird in den Feldern &lt;span style=&quot; font-weight:600;&quot;&gt;Sudname&lt;/span&gt;, &lt;span style=&quot; font-weight:600;&quot;&gt;Kommentar&lt;/span&gt;, &lt;span style=&quot; font-weight:600;&quot;&gt;Malz&lt;/span&gt;, &lt;span style=&quot; font-weight:600;&quot;&gt;Hopfen&lt;/span&gt;, &lt;span style=&quot; font-weight:600;&quot;&gt;Hefe&lt;/span&gt;, &lt;span style=&quot; font-weight:600;&quot;&gt;Zusätze&lt;/span&gt; und &lt;span style=&quot; font-weight:600;&quot;&gt;Tags&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="placeholderText">
        <string>Suche</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblFilter">
       <property name="text">
        <string notr="true">n / N</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <widget class="TableView" name="table">
      <property name="contextMenuPolicy">
       <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;In diesem Feld werden &lt;span style=&quot; font-weight:600;&quot;&gt;alle Sude&lt;/span&gt; ausgeführt und können nach &lt;span style=&quot; font-weight:600;&quot;&gt;gebraut&lt;/span&gt;, &lt;span style=&quot; font-weight:600;&quot;&gt;abgefüllt&lt;/span&gt;, &lt;span style=&quot; font-weight:600;&quot;&gt;verbraucht&lt;/span&gt; und &lt;span style=&quot; font-weight:600;&quot;&gt;gemerkt&lt;/span&gt; abgefüllt werden. Es stehen mehrere versteckte Funktionen zu Verfügung:&lt;/p&gt;&lt;p&gt;Mit &lt;span style=&quot; font-weight:600;&quot;&gt;Rechtsklick&lt;/span&gt; auf den einzelnen Sud oder die Spalte lassen sich Spalten mit diversen Parametern hinzufügen oder entfernen.&lt;/p&gt;&lt;p&gt;Ferner kann mit Rechtsklick ein markierter Sud jederzeit der Merkliste hinzugefügt sowie als nicht verbraucht markiert werden.&lt;/p&gt;&lt;p&gt;Mit &lt;span style=&quot; font-weight:600;&quot;&gt;Shift&lt;/span&gt; können alle oder mehrere Sude zusammen angewählt und ihre Mengenwerte im rechten Fensterteil ausgegeben werden.&lt;/p&gt;&lt;p&gt;Dabei wird für jede einzelne Zutat die Gesamtmenge in Benötigt, Vorhanden und Rest berechnet. Besonders hilfreich für die Sudplanung ist der Hinweis auf &lt;span style=&quot; font-weight:600;&quot;&gt;Fehlmengen&lt;/span&gt;, als &lt;span style=&quot; font-weight:600;&quot;&gt;negativer Wert &lt;/span&gt;mit pinkener Einfärbung angegeben.&lt;/p&gt;&lt;p&gt;Nicht in der Rohstofliste vorhandene Zutaten werden ausgegraut mit 0 angezeigt.&lt;/p&gt;&lt;p&gt;Alle Einträge können jetzt kopiert und ausgdruckt werden. Per Shift lassen sich auch Auszüge kopieren und ausdrucken.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="WdgWebViewEditable" name="webview" native="true"/>
    </widget>
   </item>
  </layout>
  <action name="actionAdd">
   <property name="icon">
    <iconset theme="sud_add"/>
   </property>
   <property name="text">
    <string>Anlegen</string>
   </property>
   <property name="toolTip">
    <string>Neuer Sud anlegen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="icon">
    <iconset theme="sud_import"/>
   </property>
   <property name="text">
    <string>Importieren</string>
   </property>
   <property name="toolTip">
    <string>Rezept importieren</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset theme="sud_export"/>
   </property>
   <property name="text">
    <string>Exportieren</string>
   </property>
   <property name="toolTip">
    <string>Sud exportieren</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset theme="sud_copy"/>
   </property>
   <property name="text">
    <string>Kopieren</string>
   </property>
   <property name="toolTip">
    <string>Sud kopieren</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset theme="sud_delete"/>
   </property>
   <property name="text">
    <string>Löschen</string>
   </property>
   <property name="toolTip">
    <string>Sud löschen</string>
   </property>
  </action>
  <action name="actionSplit">
   <property name="icon">
    <iconset theme="sud_split"/>
   </property>
   <property name="text">
    <string>Teilen</string>
   </property>
   <property name="toolTip">
    <string>Sud teilen</string>
   </property>
  </action>
  <action name="actionPin">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="sud_pin"/>
   </property>
   <property name="text">
    <string>Merken</string>
   </property>
   <property name="toolTip">
    <string>Sud merken</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset theme="sud_load"/>
   </property>
   <property name="text">
    <string>Laden</string>
   </property>
   <property name="toolTip">
    <string>Sud laden</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>widgets/tableview.h</header>
  </customwidget>
  <customwidget>
   <class>WdgWebViewEditable</class>
   <extends>QWidget</extends>
   <header>widgets/wdgwebvieweditable.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FilterButtonSud</class>
   <extends>QToolButton</extends>
   <header>widgets/filterbuttonsud.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btnFilter</tabstop>
  <tabstop>tbFilter</tabstop>
  <tabstop>table</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
